### PR TITLE
Bump Lingui to v4

### PR DESCRIPTION
**Pull request by bravo-kernel**
Refs #75

### CODE REVIEW

File: lingui.config.js [hunk 1]
```diff
@@ -8,10 +8,6 @@ module.exports = {
   fallbackLocales: {
     default: nextConfig.i18n.defaultLocale, // en
   },
```
Comment (thekip) [inline lingui.config.js:10]: I think you also can drop this. Lingui will always fallback to source locale while compiling

File: package.json [hunk 2]
```diff
@@ -64,10 +64,10 @@
   "devDependencies": {
     "@babel/core": "7.21.0",
     "@babel/preset-react": "7.18.6",
```
Comment (thekip) [inline package.json:66]: You can delete all this babel deps if they were used only for lingui

File: src/translations/activateTranslation.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { i18n } from "@lingui/core"
+
+i18n.loadLocaleData({
+  en : {},
+  nl: {},
+  pseudo: {},
+})
```
Comment (thekip) [inline src/translations/activateTranslation.ts:7]: You should have plural rules

Comment (bravo-kernel) [reply src/translations/activateTranslation.ts:7]: Will re-add, thought they were optional (as it did not break anything nor produced errors in console).

Comment (thekip) [reply src/translations/activateTranslation.ts]: you can use this snippet 

```ts
export function activateLocale(i18n: I18n, locale: string, messages: Messages) {
  const cardinalRules = new Intl.PluralRules(locale, { type: 'cardinal' });
  const ordinalRules = new Intl.PluralRules(locale, { type: 'ordinal' });

  i18n.loadLocaleData(locale, {
    plurals: (n, ordinal) => {
      return (ordinal ? ordinalRules : cardinalRules).select(n);
    },
  });

  i18n.load(locale, messages)
  i18n.activate(locale)
}
```
from here https://github.com/thekip/nextjs-translation-demo/blob/update-for-latest-lingui/src/utils.ts

Comment (thekip) [reply src/translations/activateTranslation.ts]: Without plural rules plurals will not work correctly in languages diffrent from english. read here https://lingui.dev/guides/plurals

**Pull request merged**